src/plugins/formatters.commerce.ts

```diff
@@ -146,13 +147,62 @@ export class ProductPriceFormatter extends Formatter {
 
 export class SubscriptionPriceFormatter extends Formatter {
```

Comment:
The SubscriptionPriceFormatter logic was ported from the corresponding [formatter](https://github.com/Squarespace/template-compiler/blob/main/core/src/main/java/com/squarespace/template/plugins/platform/CommerceFormatters.java/#L473-L533) in template-compiler